### PR TITLE
Replaces commands.json with new syntax

### PR DESCRIPTION
**Pull request by itamarhaber**
Besides pulling in the new 'commands.json', this also:

* Adds man pages for some pure container commands, such as `CONFIG` and `SCRIPT` (put differently, all containers except `COMMAND`)
* Adds man pages for commands with help, e.g. `CLIENT HELP` and `SCRIPT HELP`
* Adds man pages for system (internal) commands: `DEBUG`, `PFSELFTEST`, `PFDEBUG`, `XSETID`, `REPLCONF` and `RESTORE-ASKING`
* Adds a man page for the deprecated `SUBSTR` command
* Adds man pages for missing (deprecated) _RO commands: `GEORADIUS_RO`, `GEORADIUSBYMEMBER`
* Opportunistically fixes the wrong use of `@example` tags in some `LATENCY` man pages (should be `@examples`)

This goes hand in hand with https://github.com/redis/redis-io/pull/259

- [ ] WIP `COMMAND LIST` (#1697)
- [ ] WIP Functions

### CODE REVIEW

File: commands/substr.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Returns the substring of the string value stored at `key`, determined by the
```
Comment (oranagra) [inline commands/substr.md:1]: Missing depreciation notice?

**Review by oranagra — COMMENTED** [review]
So other than replacing the json file, this PR does:
1. Add entries for container commands (like CONFIG, DEBUG, SCRIPT, etc) 
2. Add the HELP sub commands
3. Rename "example" to "examples"
4. Add SUBSTR, GEORADIUS_RO, 
5. Anything else? 
Please list in the top comment.

File: commands/substr.md (hunk 1 above)
Comment (oranagra) [reply commands/substr.md:1]: Or maybe it's suppose to be auto generated, and we need to remove it (and other Metadata like history and complexity) from all other files?

**Comment by itamarhaber** [comment]
@oranagra added.

File: commands/substr.md (hunk 1 above)
Comment (itamarhaber) [reply commands/substr.md:1]: Exactly. I can actually make the change to redis-io to use the new deprecation metadata now, but I want to clear it from the files first. Perhaps we'll leave a deprecation notice in substr just for now?

Comment (itamarhaber) [inline commands/substr.md:1]:
```suggestion
**Note**: as of Redis 2.0, this command was renamed to `GETRANGE`.

Returns the substring of the string value stored at `key`, determined by the
```

Comment (oranagra) [reply commands/substr.md:1]: if you wish to do it in steps, ok..
but let's not forget.

**Pull request merged**